jdaviz/configs/default/plugins/viewers.py

```diff
@@ -770,8 +770,11 @@ def set_plot_axes(self):
 
         if solid_angle_unit is not None:
 
-            for un in locally_defined_flux_units:
-                locally_defined_sb_unit = un / solid_angle_unit
+            # create an equivalency for each flux unit for flux <> flux/pix2.
```

Comment:
not sure why this is in the diff, this came from my pr at one point but I modified it after some review comments to be in a loop insteads